whelk-core/src/main/groovy/whelk/util/LegacyIntegrationTools.groovy

```diff
@@ -14,7 +14,7 @@ import whelk.JsonLd
 
 @CompileStatic
 class LegacyIntegrationTools {
-    
+    // FIXME: de-KBV/Libris-ify
```

Comment:
Blocker :no_entry_sign:

Reply:
Not *fully* a blocker, as these are "only" used as symbols in the vocabulary (now "any" vocabulary, as long as the system context conforms to certain (still implicit) XL conventions).